src/backend/joanie/core/utils/payment_schedule.py

```diff
@@ -105,3 +110,30 @@ def generate(total, start_date, end_date):
     installments = _calculate_installments(total, due_dates, percentages)
 
     return installments
+
+
+@app.task
```

Comment:
According to the celery documentation, the convention is to add the tasks in a `tasks` module: https://docs.celeryq.dev/en/stable/django/first-steps-with-django.html#using-celery-with-django

```
Next, a common practice for reusable apps is to define all tasks in a separate tasks.py module, and Celery does have a way to auto-discover these modules:

app.autodiscover_tasks()

With the line above Celery will automatically discover tasks from all of your installed apps, following the tasks.py convention:
```